docs/downloads.md

```diff
@@ -0,0 +1,42 @@
+# OpenMS Installers
+
+| Platform | Name | SHA256 Hash |
+|----------|------|-------------|
+| Windows   | [OpenMS-2.8.0-Win64.exe](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/2.8.0/OpenMS-2.8.0-Win64.exe) | `d203985c7042b885ac1085c30a2d9f36d7609b47`|
+| macOS | [OpenMS-2.8.0-macOS.dmg](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/2.8.0/OpenMS-2.8.0-macOS.dmg) | `d203985c7042b885ac1085c30a2d9f36d7609b47` |
+| GNU/Linux | [OpenMS-2.8.0-Debian-Linux-x86_64.deb](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/2.8.0/OpenMS-2.8.0-Debian-Linux-x86_64.deb) | `d203985c7042b885ac1085c30a2d9f36d7609b47` |
+| Source | [OpenMS-2.8.0-src.tar.gz](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/2.8.0/OpenMS-2.8.0-src.tar.gz) | `d203985c7042b885ac1085c30a2d9f36d7609b47` |
+
+
+# Workflows
```

Comment:
this workflows section has a weird format. it is too wide for the page. although there is a scrollbar at the bottom of the table, I don't see any borders around it and it's not obvious where the table ends. it doesn't look obvious that one can scroll horizontally to read more info.
btw. I have dark mode on in Windows and see most of the page in black/grey:
![image](https://user-images.githubusercontent.com/15832022/171969115-580cf093-b3fb-4866-ad09-aebfed7f801d.png)

Reply:
it has a scroll bar at the bottom, what do you see as better ui/ux?